### DISCUSSION TITLE
suggestion: check NPM version

### DISCUSSION THREAD

**Discussion by electronstudio** [shannah/jdeploy#35] [Ideas]
Debian's NPM is out of date and I had to upgrade it as suggested in the docs before package upload would work.  The error wasn't obvious ("missing package.json") so maybe you could do a check of the NPM version and tell the user to upgrade if necessary.

**Comment by shannah** [comment]
Thanks for the suggestion.  You're right.  That error message doesn't look very helpful.   I'm not sure what the *actual* minimum npm version is, so I'll pick the version I'm using, I guess, and add a check to make sure that it is running at least that version.
